cmd/kots/cli/admin-console-push-images.go

```diff
@@ -55,9 +70,19 @@ func AdminPushImagesCmd() *cobra.Command {
 				ProgressWriter: os.Stdout,
 			}
 
-			err := kotsadm.PushImages(airgapArchive, options)
-			if err != nil {
-				return errors.Wrap(err, "failed to push images")
+			_, err := os.Stat(airgapArchive)
+			if err == nil {
+				err := kotsadm.PushImages(airgapArchive, options)
+				if err != nil {
+					return errors.Wrap(err, "failed to push images")
+				}
+			} else if os.IsNotExist(err) {
+				err := kotsadm.CopyImages(airgapArchive, options, namespace)
```

Comment:
took me a second to realize this isn't actually gonna be the airgap archive but the source registry endpoint, right?

Reply:
yeah, docker.io is the only one right now.  that's where out images are hosted